articles/data-analytics-using-pandas/index.md

```diff
@@ -0,0 +1,112 @@
+ Data science and data analytics are some of the hottest topics in the industry. The ability to analyze and make predictions based on data is nothing short of extraordinary. *Python is one of the most popular languages among the data science community due to its ease of use and rich collection of libraries built to work with data. Pandas is one such library that makes handling data easy and efficient*. In this tutorial, we are going to understand how Pandas can be used to explore and draw insights from data. 
+
+### Table of Contents	 
+  
+ [Prerequisites](#prerequisites)
+ [What is Pandas?](#what-is-pandas)	
+ [Analysing data using Pandas](#analyzing-data-using-pandas)	 
+ [Code](#code)	
+ [Conclusion and further reading](#conclusion-and-further-reading)	 
+ 
+### Prerequisites
+1. A Basic understanding of programming in [Python](https://www.learnpython.org/)
+2. Basic knowledge of [data analytics](https://data-flair.training/blogs/data-analytics-tutorial/)
+
+### What is Pandas?
+According to the [official documentation](https://pandas.pydata.org/docs/),  pandas is a fast, powerful, flexible, and easy to use open-source data analysis and manipulation tool, built on top of the Python programming language. Pandas is usually used in conjunction with Jupyter notebooks, making it more powerful and efficient for [exploratory data analysis](https://towardsdatascience.com/exploratory-data-analysis-in-python-c9a77dfa39ce). If you are new to Jupyter notebooks, [this](https://www.section.io/engineering-education/introduction-to-jupyter-notebooks/) article walks you through the installation and basics of Jupyter notebooks. 
```

Comment:
> According to the [official documentation](https://pandas.pydata.org/docs/),  pandas is a fast, powerful, flexible, and easy to use open-source data analysis and manipulation tool, built on top of the Python programming language.

This sentence is too hard to read. Could be split into smaller, easier-to-read sentences.

For example, 

According to the official documentation, pandas is a fast, powerful, flexible, and easy to use open-source data analysis and manipulation tool. It is built on top of the Python programming language.